py3status/modules/keyboard_locks.py

```diff
@@ -7,8 +7,8 @@
     format: display format for this module (default '{caps} {num} {scr}')
     icon_caps_off: show when Caps Lock is off (default 'CAPS')
     icon_caps_on: show when Caps Lock is on (default 'CAPS')
-    icon_num_off: show when Num Lock is off (default 'NUM')
-    icon_num_on: show when Num Lock is off (default 'NUM')
+    icon_num_off: show when Numeric Lock is off (default 'NUM')
+    icon_num_on: show when Numeric Lock is on (default 'NUM')
```

Comment:
I want to keep out abbreviations in description. See `Scroll Lock` for scr. Now I ponder if I should fix caps too really quick: `show when Capitals Locks is on (default 'CAPS')`: to make it slightly easier to parse. 

We're just too used to `Num Lock` abbreviations, is all. IHMO. By the way, we should look into async one day to see if we can listen for `locks` as not to call this module every 1s.